src/libs/kata-types/tests/test-config.rs

```diff
@@ -0,0 +1,456 @@
+// Copyright (c) 2019-2021 Alibaba Cloud
```

Comment:
I think the file name should use underscores like all other source files.

Reply:
fixed